mmdet/models/utils/csp_layer.py

```diff
@@ -0,0 +1,149 @@
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, DepthwiseSeparableConvModule
+from mmcv.runner import BaseModule
+
+
+class DarknetBottleneck(BaseModule):
+    """The basic bottleneck block used in Darknet.
+
+    Each ResBlock consists of two ConvModules and the input is added to the
+    final output. Each ConvModule is composed of Conv, BN, and LeakyReLU.
+    The first convLayer has filter size of 1x1 and the second one has the
+    filter size of 3x3.
+
+    Args:
+        in_channels (int): The input channels of this Module.
+        out_channels (int): The output channels of this Module.
+        expansion (int): The kernel size of the convolution. Default: 0.5
+        with_res_shortcut (bool): Whether to use residual shortcut.
```

Comment:
with_res_shortcut -> with_residual

Reply:
Seem it is identity, I rename it to `add_identity` which is used in
https://github.com/open-mmlab/mmcv/blob/04daea425bcb0a104d8b4acbbc16bd31304cf168/mmcv/cnn/bricks/transformer.py#L281